proposals/0001-template.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Extract WebView From Core
+author:
+- Jamon Holmgren
+date: 2018-07-30
+---
+
+# RFC0000: Extract WebView From Core
+
+## Summary
+
+Extract the current WebView implementation from React Native Core and move it into a third party package.
+
+## Basic example
+
+Currently, WebView is included in React Native core:
+
+```jsx
+import { WebView } from 'react-native';
+```
+
+This would move it to an external package, like so:
+
+```jsx
+import { WebView } from '@infinitered/react-native-webview';
+```
+
+## Motivation
+
+The current WebView implementation is (to my knowledge) not used by Facebook internally. This leads (understandably) to a lack of first-class support for this core API. By moving the component to a third party that is more invested in supporting it, there will be better support.
+
+Additionally, with this component no longer included in core, there will be an opening for alternative third party WebView implementations, should anyone else wish to provide one.
+
+## Detailed design
+
+For the first release, the goal is to provide the same API, only replacing the import statement and otherwise backwards compatible.
+
+We will also want to update generated project boilerplates, such as create-react-native-app and Ignite, to use the new package.
```

Comment:
Should we do it in the first add more features/bigs and then look at a replacement ?

Reply:
I'm definitely open to that. I mainly wanted to keep the surface area of the change smaller for those who have to migrate their existing apps.